### PR TITLE
finished seed script/db setup

### PR DESCRIPTION
**Pull request by robeusanio11**

### CODE REVIEW

**Review by ndw001 — APPROVED** [review]

File: database/Model.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    }
+  ],
+  timestamps: false
+});
+
+// Defines answers table
+const Answers = db.define('answers', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
```
Comment (dreamsmasher) [inline database/Model.js:78]: maybe abstract some of these common denominators into an object to reduce code size

File: database/seed.js [hunk 2]
```diff
@@ -1,55 +1,227 @@
...
+// pool.connectAsync()
+//   .catch((err) => { console.error(err); });
+
+const seed = async () => {
+
+  let start = performance.now();
+
+  await Answers.drop();
+  await Questions.drop();
+  await Users.drop();
```
Comment (dreamsmasher) [inline database/seed.js:33]: this should be a loop - 

`await Promise.all([obj1, obj2 ...].map(async obj => {await obj.drop(); await obj.sync()}))`

File: database/seed.js [hunk 3]
```diff
@@ -1,55 +1,227 @@
...
+          id: i,
+          name: faker.random.words(),
+        });
+      }
+      // Writes list of records to a CSV
+      await ItemsCsvWriter.writeRecords(list)
+        .catch((err) => { console.error(err) })
+
+      // Copies CSV file to DB
+      db.query(`COPY items ("id", "name") FROM '/home/robeusanio11/minh-services/database/CSVs/items${j}.csv' DELIMITER ',' CSV HEADER;`)
```
Comment (dreamsmasher) [inline database/seed.js:70]: use __dirname to prevent posting your personal info to github

File: database/seed.js [hunk 4]
```diff
@@ -1,55 +1,227 @@
...
+
+    // Will create 5,000,000 total records
+    for (var j = 0; j < 5; j++) {
+
+      // Defines rules for QuestionsCsvWriter
+      const QuestionsCsvWriter = createObjectCsvWriter({
+        path: `./database/CSVs/questions${j}.csv`,
+        header: [
+          { id: 'id', title: 'id' },
+          { id: 'itemId', title: 'itemId' },
```
Comment (dreamsmasher) [inline database/seed.js:130]: maybe use a loop for this too

File: database/seed.js [hunk 5]
```diff
@@ -1,55 +1,227 @@
...
+    var max = 1000000;
+
+    // Will create 50,000,000 records
+    for (var j = 0; j < 50; j++) {
+
+      // Defines rules for AnswersCsvWriter
+      var AnswersCsvWriter = createObjectCsvWriter({
+        path: `./database/CSVs/answers${j}.csv`,
+        header: [
+          { id: 'id', title: 'id' },
```
Comment (dreamsmasher) [inline database/seed.js:180]: r e p e t i t i o n

File: database/seed.js [hunk 6]
```diff
@@ -1,55 +1,227 @@
...
+  await createQuestionsObjectArray();
+  await createAnswersObjectArray();
 
-let insertData = () => {
-  Question.create(data)
-    .then(() => db.close())
-    .catch(err => console.log(err));
-};
+  let end = performance.now()
+  console.log(`it took ${Math.floor((end - start)/1000)} seconds to create and seed all data`);
```
Comment (dreamsmasher) [inline database/seed.js:224]: `console.time()` is a bit more convenient for this kind of stuff

File: server/index.js [hunk 7]
```diff
@@ -1,31 +1,16 @@
 const express = require('express');
+const app = express();
 const path = require('path');
 const bodyParser = require('body-parser');
-const Question = require('../database/question.js');
 const cors = require('cors');
-const app = express();
 
-const port = 1337;
+const PORT = 1337;
```
Comment (dreamsmasher) [inline server/index.js:7]: nice

**Review by dreamsmasher — COMMENTED** [review]
Pretty clean, but there are some parts that could be abstracted away for readability and clarity. DRY

**Pull request merged**